tests/onboarding/test_onboarding_import_seed.py

```diff
@@ -7,7 +9,8 @@
 from driver.aut import AUT
 from gui.components.onboarding.before_started_popup import BeforeStartedPopUp
 from gui.components.onboarding.beta_consent_popup import BetaConsentPopup
-from gui.components.splash_screen import SplashScreen
+from gui.components.splash_screen_did_u_know import SplashScreenDidYouKnow
```

Comment:
No need to import SplashScreenDidYouKnow ? Don't see it in test

Reply:
yes, sure, i forgot, sorry about that